docs/styles.md

```diff
@@ -26,16 +26,20 @@
 * Layout
   - Flex
     - Initialization of Flex Box
-      - [Required Flex Class](#side-by-side-os-raise-d-flex)
+      - [Required Flex Class](#side-by-side-os-raise-d-flex-nowrap)
```

Comment:
I feel like we need to think through the doc changes a bit more in terms of structure. We've got two classes that can be used for flex box, but we now only document one (the other is referenced indirectly in other places like the table styles). This ToC indicates to me that this section is layout agnostic, but it takes me specifically to `Side by side (os-raise-d-flex-nowrap)`. There are probably cases when we want to use one vs the other, and we should help ensure that is made clear for authors.

On a related note: We have quite a bit of content already that is using `os-raise-d-flex` for "side-by-side". Unfortunately, those likely entail both intended scenarios (e.g. where we want wrapping and where don't), so there may not be a perfect solution in terms of being "backwards compatible". However, we may want to think things through a bit as far as the impact of effectively changing what we are using moving forward vs in the past.

Reply:
Just a quick follow-up (for discussion): Maybe we need to just carefully document uses of both `os-raise-dl-flex` and `os-raise-d-flex-nowrap` since some of this will be up to what behavior the CM wants. The table scenarios we have are a case in point, but I think in general the flex box section should be self-consistent + holistic in how it describes the various utility classes. :thinking:

Reply:
My original thought was to document both `os-raise-d-flex` and `os-raise-d-flex-nowrap`. Once those two are established, then the next "step" would be to show how each of our utility classes work when paired with `os-raise-d-flex` and `os-raise-d-flex-nowrap`. If the behavior was the same, then we could state as much. If the behavior was different then we can show / explain said different behavior.

I also thought it might be particularly helpful to explicitly state that utility classes "A - F" will only work with one of the flex classes. It may also be useful to note that some utility classes work on the flex parent, while others only work on the flex "children" (aka the flex-items).

Reply:
Documentation has been updated. There are now individual examples for each item as it relates to flexbox. If there is anything that doesn't seem to flow, let me know. Overall, I think this will hopefully provide more clarity on flexbox usage.

Reply:
Thanks for the doc improvements, @MReyna12 ! They look great to me.